doc/CONFIGURATION.md

```diff
@@ -128,11 +127,9 @@ Some available peer discovery modules are:
 - [@libp2p/mdns](https://github.com/libp2p/js-libp2p-mdns)
 - [@libp2p/bootstrap](https://github.com/libp2p/js-libp2p-bootstrap)
 - [@libp2p/kad-dht](https://github.com/libp2p/js-libp2p-kad-dht)
-- [@libp2p/webrtc-star](https://github.com/libp2p/js-libp2p-webrtc-star)
+- [@libp2p/webrtc-star](https://github.com/libp2p/js-libp2p-webrtc-star) ([Archived](https://github.com/libp2p/js-libp2p-webrtc-star/blob/master/README.md#%EF%B8%8F%EF%B8%8F%EF%B8%8F%EF%B8%8F%EF%B8%8F%EF%B8%8F))
 - [@chainsafe/discv5](https://github.com/chainsafe/discv5)
 
-**Note**: `peer-discovery` services within transports (such as `js-libp2p-webrtc-star`) are automatically gathered from the `transport`, via it's `discovery` property. As such, they do not need to be added in the discovery modules. However, these transports can also be configured and disabled as the other ones.
```

Comment:
This is still true of modules like `kad-dht` so might be worth mentioning here.

Reply:
I think the note is a bit confusing overall, since we configure our discovery service via a services module now, intuitively one would configure that separately from the transport. 

So I think it's best to exclude this note.